content/renderer/media/media_factory.cc

```diff
@@ -144,6 +144,17 @@ namespace {
 // to be many of those. See http://crbug.com/1232649
 constexpr size_t kDefaultMaxWebMediaPlayers = 1000;
 
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+// The following variables match |kSbPlayerWriteDurationLocal|
+// and |kSbplayerWriteDurationRemote| in //starboard/player.h.
+//
+// The audio write duration when all the audio connectors are local.
+constexpr int kWriteDurationLocal = 1000000 / 2;  // 0.5 seconds
```

Comment:
nit: consider to make these two numbers using `base::Milliseconds(500)` and `base::Seconds(10)`, so we don't need to add comments like `// 0.5 seconds` after them, as it is self-contained.